runtime/compiler/runtime/JITServerIProfiler.cpp

```diff
@@ -538,7 +539,10 @@ JITServerIProfiler::setCallCount(TR_OpaqueMethodBlock *method, int32_t bcIndex,
             // Info for this bcIndex is missing. 
             // Create a new entry, add it to the cache and send a remote message as well
             uintptrj_t methodStart = TR::Compiler->mtd.bytecodeStart(method);
-            TR_IPBCDataCallGraph *entry = new TR_IPBCDataCallGraph(methodStart + bcIndex);
+            TR_AllocationKind allocKind = methodInfoPresentInPersistent ? persistentAlloc : heapAlloc;
+            TR_IPBCDataCallGraph *entry = (TR_IPBCDataCallGraph*)comp->trMemory()->allocateMemory(sizeof(TR_IPBCDataCallGraph), allocKind, TR_Memory::IPBCDataCallGraph);
```

Comment:
We have `TR_IPBytecodeHashTableEntry *entry` above and `TR_IPBCDataCallGraph *entry` here and they overlap in scope. It took me a few moments to figure out, but maybe we should use different variable names because having the same name doesn't seem to be adding any clarity.

Reply:
Updated